RecoVertex/BeamSpotProducer/src/PVFitter.cc

```diff
@@ -87,6 +86,8 @@ void PVFitter::initialize(const edm::ParameterSet& iConfig,
   errorScale_        = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("errorScale");
   sigmaCut_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("nSigmaCut");
   fFitPerBunchCrossing=iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("FitPerBunchCrossing");
+  useOnlyFirstPV_    = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("useOnlyFirstPV");
+  minSumPt_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("minSumPt");
```

Comment:
Why double, if sumPt is a float? You should better go through the list and check which parameters could profit of a different type.

Why are all these parameter "untracked"? You should better go through the list and check which parameters must be tracked, and which untracked.

Commented out lines of code should be either removed (in the whole class implementation), or accompanied by a comment line which explains why do you want to keep them in the code.

Where are deleted the hPVx and hPVy TH2F right below?